bindings/cs/MongoDB.Libmongocrypt/CryptClientFactory.cs

```diff
@@ -46,7 +46,6 @@ public static CryptClient Create(CryptOptions options)
             // to avoid restrictions on target frameworks that present in some of below
             if (OperatingSystemHelper.CurrentOperatingSystem != OperatingSystemPlatform.Windows)
             {
-#if !NETSTANDARD1_5
```

Comment:
now, we won't protect our code from throwing not supported exceptions for TF `netstandard1.5` (and < netstandard2.1 for GCP). When MONGOCRYPT-257 will be resolved, the C level will be able to fully correct handle these cases